index.html

```diff
@@ -446,17 +446,25 @@
+
+   - the coverage (presence) weight of the film, **`thin_film_weight`**,
+   - the thickness of the film in nanometers, **`thin_film_thickness`**,
+   - the index of refraction (IOR) of the film, **`thin_film_ior`**,
+
+In practice, this wave-optics effect is most easily incorporated directly into the Fresnel factor of the microfacet BSDFs of both the metal and dielectric-base layers. The currently recommended thin-film model is that of Belcour and Barla [#Belcour2017]. The coverage weight functions as a blend between the BSDF with and without the presence of the film, and is necessary since neither the thickness nor the IOR of the film provide a dial of the effect without altering the shape and saturation of the color fringes.
 
+The interference within and resulting reflection from the film will be affected (as described by Belcour and Barla [#Belcour2017]) by the complex IOR of the adjacent media above and below the film, which in general are a statistical mix of metal and dielectric below and coat and ambient medium above. Modeling of the precise effect is implementation-dependent, however as described in the Metal section the Fresnel factor for metal is defined according to the Schlick-based "F82-tint" parametrization, which does not specify the underlying (complex) physical IOR. We suggest here that some reasonable approximation is employed to map the Fresnel factor to effective IOR and absorption coefficients, for example that described by [#Gulbrandsen2014].
 
-Thin-film params          | Label     | Type     | Range           | Norm          | Default | Description
---------------------------|-----------|----------|:---------------:|:-------------:|:-------:|----------------------------------------------
-**`thin_film_thickness`** | Thickness | `float`  | $ [0, \infty) $ | $ [0, 2000] $ | $ 0   $ | Thickness of the film in nanometers
-**`thin_film_ior`**       | IOR       | `float`  | $ (0, \infty) $ | $ [1, 3]    $ | $ 1.5 $ | Refractive index of the film
+
+
+Thin-film params          | Label     | Type     | Range           | Norm          | Default  | Description
+--------------------------|-----------|----------|:---------------:|:-------------:|:--------:|----------------------------------------------
+**`thin_film_weight`**    | Weight    | `float`  | $ [0, 1]      $ |               | $ 0    $ | Coverage weight of the thin film
+**`thin_film_thickness`** | Thickness | `float`  | $ [0, \infty) $ | $ [0, 2000] $ | $ 1000 $ | Thickness of the film in nanometers
+**`thin_film_ior`**       | IOR       | `float`  | $ (0, \infty) $ | $ [1, 3]    $ | $ 1.5 $  | Refractive index of the film
```

Comment:
I like @sharktacos's proposal (#150) to have a normalised parameter. This can be done in a separate PR though if there isn't consensus there.

Reply:
I think you could probably get away making the width be in micrometers, with a soft-max at 1.  So have it default to [0,1] range.

In the Belcour & Barla paper, they use the parameter $\mathcal{D} = 2 \eta d$ where $\eta$ is IOR, $d$ is width. In their tests, $\mathcal{D}$ seems to vary in range [0, 3200],  though with maximal effect at the mid-range i.e. 1600nm. So that implies even if $d$ is (soft) capped at 1 micrometer, the useful range is captured.

Reply:
Yup, sounds good.